src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -45,10 +62,28 @@ public ModelManager(ReadOnlyContactList contactList, ReadOnlyUserPrefs userPrefs
         this.contactList = new ContactList(contactList);
         this.taskList = new PriorityTaskList(taskList);
         this.userPrefs = new UserPrefs(userPrefs);
+        this.moduleList = new UniqueModuleList();
         filteredPersons = new FilteredList<>(this.contactList.getPersonList());
         this.versionedContents = new VersionedContents(new Content(getContactList(), getTaskList()));
     }
 
+    /**
+     * Initializes a ModelManager with the given contactList, userPrefs , taskList and an empty moduleList.
+     */
+    public ModelManager(ReadOnlyContactList contactList, ReadOnlyUserPrefs userPrefs,
+            ReadOnlyTaskList taskList, UniqueModuleList moduleList) {
+        requireAllNonNull(contactList, userPrefs, taskList);
```

Comment:
Similarly, might be a good idea to include moduleList here if it should not be null